esmvaltool/cmorizers/data/cmor_config/ERA-Interim.yml

```diff
@@ -126,25 +126,56 @@ variables:
     mip: day
     raw: z
     file: 'ERA-Interim_z_daily_*.nc'
+
   # monthly frequency
   clivi:
     mip: Amon
-    raw: p57.162
-    file: 'ERA-Interim_p57.162_monthly_*.nc'
+    raw: viiw
+    file: 'ERA-Interim_viiw_monthly_*.nc'
   clt_month:
     short_name: clt
     mip: Amon
     raw: tcc
     file: 'ERA-Interim_tcc_monthly_*.nc'
+#  clt-low:
+#    short_name: clt-low
+#    mip: Amon
+#    raw: lcc
+#    file: 'ERA-Interim_lcc_monthly_*.nc'
+#  clt-med:
+#    short_name: clt-med
+#    mip: Amon
+#    raw: mcc
+#    file: 'ERA-Interim_mcc_monthly_*.nc'
+#  clt-high:
+#    short_name: clt-high
+#    mip: Amon
+#    raw: hcc
+#    file: 'ERA-Interim_hcc_monthly_*.nc'
```

Comment:
Should these lines be removed from the config file or why are they commented out?

Reply:
These lines are commented out as they are currently not used (but could be in the future). If that is confusing, I can delete these lines.